### ISSUE TITLE
image keeps blinking

### ISSUE THREAD

**Issue by d0razi** [cotes2020/jekyll-theme-chirpy#1151]
https://d0razi.github.io/

When I go to the above site, the image is completely loaded, but it keeps blinking, so how do I solve it? Can we get that blinking effect out of the source code?

**Comment by d0razi** [comment]
It just happened to be solved. I think it was solved by adding the toc:false option.

**Comment by ciphermenial** [comment]
This is fixed in the latest version.

**Comment by jefflongo** [comment]
I don't think this is fixed? If I remove this line from the current master branch
https://github.com/cotes2020/jekyll-theme-chirpy/blob/37c976499ead51c1d88e8e8213366240a72adebc/_posts/2019-08-08-text-and-typography.md?plain=1#L12
it will shimmer forever.

**Comment by ciphermenial** [comment]
> EDIT: after deploying it seems to work, so it must be some sort issue with my local environment setup?

That is a possibility. After updating to the latest version on my github pages, the shimmering stopped.
